src/bundle/Resources/views/RichText/fields/ezrichtext/custom_tags/ezfacebook.html.twig

```diff
@@ -0,0 +1,24 @@
+{% if align is defined %}
+    {% if align == 'left' %}
+        {% set justify_content = 'flex-start' %}
+    {% elseif align == 'right' %}
+        {% set justify_content = 'flex-end' %}
+    {% elseif align == 'center' %}
+        {% set justify_content = 'center' %}
+    {% endif %}
+{% endif %}
+<div
+    class="fb-post"
+    style="display: flex;{% if justify_content is defined %} justify-content: {{ justify_content }};{% endif %}"
```

Comment:
please move css to css/scss files

Reply:
I'm not so sure about that.
As far as i know, this template goes to the "frontend", and can be loaded multiple times in one page. If we include here CSS file, the CSS will be loaded multiple times. 
We could use webpack encore to prevent this but should we use encore in "frontend" templates?

Reply:
> I'm not so sure about that.
> As far as i know, this template goes to the "frontend", and can be loaded multiple times in one page. If we include here CSS file, the CSS will be loaded multiple times.

We definitely don't want that.

> We could use webpack encore to prevent this but should we use encore in "frontend" templates?

If it's possible to do it in a self-contained way, so you'll add some encore Twig function and some (s)css asset in this bundle which will work out-of-the-box then maybe we should go for that. If doing this requires extra configuration either by end-developer or in meta, then we should leave it as is now.